### PR TITLE
recon-util: added skip option

### PR DESCRIPTION
**Pull request by raffaelladevita**

### CODE REVIEW

**Comment by baltzell** [comment]
checking feedback issues, again

**Comment by baltzell** [comment]
checking feedback issues, again

File: common-tools/clas-reco/src/main/java/org/jlab/clas/reco/EngineProcessor.java [hunk 1]
```diff
@@ -272,11 +272,15 @@ public void processFile(String file, String output, int nevents){
...
             writer.open(output);
 
+            if(nskip>0 && nevents>0) nevents += nskip;
+            
             ProgressPrintout  progress = new ProgressPrintout();
             while(reader.hasEvent()==true){
                 DataEvent event = reader.getNextEvent();
-                processEvent(event);
-                writer.writeEvent(event);
+                if(eventCounter>nskip || nskip<0) {
```
Comment (baltzell) [inline common-tools/clas-reco/src/main/java/org/jlab/clas/reco/EngineProcessor.java]: Shouldn't this be nskip <= 0 ?

Comment (raffaelladevita) [reply common-tools/clas-reco/src/main/java/org/jlab/clas/reco/EngineProcessor.java]: That's right: it's done in the latest commit

**Review by baltzell — APPROVED** [review]

**Pull request merged**